mlrun/utils/regex.py

```diff
@@ -30,6 +30,8 @@
     r"^[a-z0-9]([-a-z0-9]*[a-z0-9])?$",
 ]
 
+tag_name = [r"^[^!@$%^&*()_+#]+$"]
```

Comment:
This is the list of "forbidden chars" as provided in the bug, but I think it was just an example, and it would be better to use a white-list of allowed chars instead - I'm not sure at all that this list is full. Also, it seems funny to me that we don't allow `_` in tags, but we do allow things like `;` or `~`...

Reply:
Would something like [A-Za-z0-9.-] be better?